apps/keyboard/js/views/swipeable_panel_view.js

```diff
@@ -110,100 +122,93 @@ SwipeablePanelView.prototype._updateIndicator = function() {
 };
 
 SwipeablePanelView.prototype.handleEvent = function(evt) {
-  var totalWidth;
-
   switch (evt.type) {
     case 'touchstart':
-      this.touchStartTimestamp = evt.timeStamp;
       this.startX = evt.touches[0].pageX;
       this.deltaX = 0;
```

Comment:
`touchstart` should be abstracted in `SwipingDetector`; `SwipeablePanelView` should not be listening and keeping these properties.